plugins/systemd/systemd_units

```diff
@@ -123,8 +125,12 @@ fetch () {
 		count=$(echo "$state_unit_list" | grep -c "$state$")
 		echo "$state.value $count"
 		extinfo=$(echo "$state_unit_list" | grep "$state$" | cut -d " " -f 1 | tr '\n' ' ')
-		if [ -n "$extinfo" ]; then
-			echo "$state.extinfo" "$extinfo"
+		if [ -z "$silence_active_extinfo" ]; then
```

Comment:
I guess, you should use `[ "${silence_active_extinfo:-0}" = "0" ]` instead. Otherwise you would surprise users by misinterpreting a value of `0`.

Additionally you should set this value in the header of the plugin in order to give a quick overview, which environment variables are parsed.

Reply:
Done that.